app/scripts/components/Header/header-style.scss

```diff
@@ -15,4 +16,24 @@
       transform: translate(0, 0);
     }
   }
+
+  .header-tab {
+    height: $small-margin * 2;
+    position: relative;
+
+    &.-active {
+
+      &:after {
+        position: absolute;
+        content: '';
+        width: 22px;
+        height: 4px;
+        background-color: $theme-accent;
+        left: calc(50% - 11px);
```

Comment:
As long as you don't want to animate it I'll accept this. If you want to center it's easier to use `left: 50%; transform: translate(-50%,0)`.